crates/oxc_linter/src/rules/oxc/only_used_in_recursion.rs

```diff
@@ -184,6 +242,52 @@ fn is_function_maybe_reassigned<'a>(
     is_maybe_reassigned
 }
 
+fn is_function_exported(
```

Comment:
Look like we can use [`is_exported`](https://github.com/oxc-project/oxc/blob/c53c210efc0274ed9e3ea530e8c9606c1794fc01/crates/oxc_linter/src/rules/eslint/no_unused_vars/symbol.rs#L163).

---

Should we catch this case?

```js
 function foo(a) { foo(a) }; let bar = foo; export  { bar } 
```